Libraries/Components/WebView/WebView.ios.js

```diff
@@ -487,7 +487,13 @@ class WebView extends React.Component {
       );
       shouldStart = shouldStart && passesWhitelist;
       if (!passesWhitelist) {
-        Linking.openURL(url);
+        Linking.openURL(url)
+          .then(() => {
+            //ignored
+          })
+          .catch(() => {
+            //ignored
```

Comment:
Shouldn't we at least YellowBox in dev in this case?

Reply:
I suppose that could be handy. How do would you like the yellow box shown here?